### ISSUE TITLE
 Error: java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory 

### ISSUE THREAD

**Issue by arojascastro** [quinnanya/dlcl204#1]
Hello,

I am trying your tutorial on Stanford NLP for Spanish texts. I downloaded the package, prepared my text and run the tagger with `java -cp "*" edu.stanford.nlp.tagger.maxent.MaxentTagger -model models/spanish-distsim.tagger -textFile Lope_circe.txt -outputFormat tsv -outputFile yourposoutput.txt` (where `Lope_circe.txt` is the name of my file).

However, I got this error message 

```
Loading default properties from tagger models/spanish-distsim.tagger
Exception in thread "main" java.lang.NoClassDefFoundError: org/slf4j/LoggerFactory
	at edu.stanford.nlp.io.IOUtils.<clinit>(IOUtils.java:42)
	at edu.stanford.nlp.tagger.maxent.TaggerConfig.<init>(TaggerConfig.java:146)
	at edu.stanford.nlp.tagger.maxent.TaggerConfig.<init>(TaggerConfig.java:128)
	at edu.stanford.nlp.tagger.maxent.MaxentTagger.main(MaxentTagger.java:1836)
Caused by: java.lang.ClassNotFoundException: org.slf4j.LoggerFactory
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(BuiltinClassLoader.java:583)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(ClassLoaders.java:178)
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:521)
	... 4 more
```
I google this error and some pages recommend downloading SLF4J implementation and add more .jar files to the lib directory. I tried that and still did not work. Do you have any idea about how to fix this problem? 

Thank you in advance.

**Comment by quinnanya** [comment]
I think what happened is that you downloaded the "basic English Stanford Tagger version 3.6.0 [24 MB]" rather than the "full Stanford Tagger version 3.6.0 [124 MB]". (It's an easy mistake to make -- the basic English one is listed first). When I tried to run the command from within the basic English tagger, I got the same error. Let me know if that fixes it?

I'll adjust the wording in the tutorial to make it clearer which version to use.

**Comment by arojascastro** [comment]
Thank you for your answer! I downloaded the package again to make sure myself and I run the command but I had no success. So I went to the *FAQ* page (I should have visited before) and I found this: 

> Why do I get Exception in thread "main" java.lang.NoClassDefFoundError:edu/stanford/nlp/tagger/maxent/MaxentTagger? This means your Java CLASSPATH isn't set correctly, so the tagger (in stanford-tagger.jar) isn't being found. See the examples in the README.txt file for how to set the classpath with the -cp or -classpath option. See, for example, http://en.wikipedia.org/wiki/Classpath_(Java) for general discussion of the Java classpath.

Then I read the README file and I found the following command:

`java -mx300m -classpath stanford-postagger.jar:lib/* edu.stanford.nlp.tagger.maxent.MaxentTagger -model models/spanish-distsim.tagger -textFile myoutput.txt > output.txt`

or displayed as tabular data:

`java -mx300m -classpath stanford-postagger.jar:lib/* edu.stanford.nlp.tagger.maxent.MaxentTagger -model models/spanish-distsim.tagger -textFile Lope_circe.txt -outputFormat tsv -outputFile output.txt`

This is working for me. 

Thank you again! I am closing this.
